go/runner.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	inFile := os.Args[1]
+	outFile := os.Args[2]
+	fmt.Printf("Reading from: %v.  Writing to: %v\n", inFile, outFile)
+	dat, err := ioutil.ReadFile(inFile)
+	if err != nil {
+		panic(err)
```

Comment:
~ I will generally keep my `main()` to just a few lines, and move all the logic to a run, so that errors can be handled consistently without printing a stack trace.

```go
func main() {
    if err := run(); err != nil {
        fmt.Fprintf(os.Stderr, "Error: %s\n", err)
        os.Exit(1)
    }
}
```